app/src/main/java/com/sun/tino/hottrailers/ui/movie_detail/info/MovieInfoFragment.java

```diff
@@ -5,19 +5,24 @@
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.Toast;
+
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.databinding.DataBindingUtil;
 import androidx.fragment.app.Fragment;
+
 import com.sun.tino.hottrailers.R;
 import com.sun.tino.hottrailers.data.model.Genre;
 import com.sun.tino.hottrailers.databinding.FragmentMovieInfoBinding;
 import com.sun.tino.hottrailers.ui.home.adapters.GenreAdapter;
 import com.sun.tino.hottrailers.ui.movie_detail.MovieDetailViewModel;
 
-public class MovieInfoFragment extends Fragment implements GenreAdapter.GenreListener {
+public class MovieInfoFragment extends Fragment implements GenreAdapter.GenreListener,
+        MovieDetailViewModel.OnFavoriteListener {
     private FragmentMovieInfoBinding mBinding;
     private MovieDetailViewModel mViewModel;
+    private static final String ADDED = "Added to Favorite";
```

Comment:
code conventions 2.8 về thứ tự các trường trong class: https://github.com/framgia/coding-standards/blob/master/vn/android/codingstyleguide.md#28-th%E1%BB%A9-t%E1%BB%B1-member-trong-class
Constants ở trên đầu, ngoài ra: 
```suggestion
    private static final String MSG_ADDED = "Added to Favorite";
```